examples/js/SkyShader.js

```diff
@@ -47,17 +47,18 @@ THREE.ShaderLib[ 'sky' ] = {
 		"const float e = 2.71828182845904523536028747135266249775724709369995957;",
 		"const float pi = 3.141592653589793238462643383279502884197169;",
 
+                // wavelength of used primaries, according to preetham
+		"const vec3 lambda = vec3(680E-9, 550E-9, 450E-9);",
+		
+                // this pre-calcuation replaces the older totalRayleigh(vec3 lambda) function which was:
+		// return (8.0 * pow(pi, 3.0) * pow(pow(n, 2.0) - 1.0, 2.0) * (6.0 + 3.0 * pn)) / (3.0 * N * pow(lambda, vec3(4.0)) * (6.0 - 7.0 * pn));
```

Comment:
These constants don't quite match what used to be `simplifiedRayleigh`. What are `n` and `pn` in the function you used to calculate your constants?